index.html

```diff
@@ -13461,6 +13453,83 @@ <h2>Including Elements in the Accessibility Tree</h2>
             </li>
           </ul>
 	</section>
+	<section id="tree_relationships">
+	  <h2>Relationships in the Accessibility Tree</h2>
+          <p>The following terms are used to describe relationships between <abbr title="Document Object Model">DOM</abbr> elements.</p>
+          <p>The <dfn data-export="" data-lt="accessibility child|owned child|child element|child|children|child elements">accessibility children</dfn> of a DOM element are all of the children of that element's corresponding <a>accessible object</a> in the <a>accessibility tree</a>. In terms of the DOM, that includes the following:</p>
+          <ul>
+            <li>The <abbr title="Document Object Model">DOM</abbr> children of the <a>element</a>.
+            </li>
+            <li>All DOM descendants of the <a>element</a> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.
+            </li>
+            <li>All DOM elements specified via an <pref>aria-owns</pref> relationship to the element.
+            </li>
+            <li>All DOM descendants of an element with role <rref>generic</rref> or <rref>none</rref> specified via <pref>aria-owns</pref> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.
+            </li>
+          </ul>
+          <p>And excludes the following:
+          <ul>
+            <li>All DOM elements that have no corresponding <a>accessible object</a> because they have been <a href="#tree_exclusion">excluded from the accessibility tree</a>./li>
+            <li>All DOM elements whose corresponding <a>accessible object</a> have been reparented in the <a>accessibility tree</a> via <pref>aria-owns</pref>./li>
+          </ul>
+          </p>
+          <p>In the following example, the <rref>list</rref> element has four accessibility children:</p>
+	  <pre class="example highlight">
+&lt;div role="list" aria-owns="child3 child4"&gt;
+  &lt;div role="listitem"&gt;Accessibility Child 1&lt;/div&gt;
+  &lt;div&gt;
+    &lt;div role="listitem"&gt;Accessibility Child 2&lt;/div&gt;
+  &lt;/div&gt;
+&lt;/div&gt;
+&lt;div id="child3" role="listitem"&gt;Accessibility Child 3&lt;/div&gt;
+&lt;div id="child4"&gt;
+  &lt;div role="listitem"&gt;Accessibility Child 4&lt;/div&gt;
+&lt;/div&gt;
+          </pre>
+          <p>In the following example, the <rref>list</rref> element has no accessibility children:</p>
+	  <pre class="example highlight">
+&lt;div role="list"&gt;
+  &lt;div role="listitem" aria-hidden="true"&gt;Excluded element&lt;/div&gt;
+  &lt;div role="listitem" id="reparented"&gt;Reparented element&lt;/div&gt;
```

Comment:
```suggestion
  &lt;div role="listitem" id="reparented"&gt;Reparented element&lt;/div&gt;
  &lt;div role="group"&gt;
    &lt;div role="listitem"&gt;Descendent with non-generic intervening element&lt;/div&gt;
  &lt;/div&gt;
```
How do you feel about clarifying the non-generic-intervening element thing as well? (I know `group` isn't technically an allowed child, so it's maybe a slightly weird example)

Reply:
Hmmm that is a good point... but I feel like I would want a separate example for that. Because in this example it says (for simplicity) this list item has no accessibility children. We could have an example with an "accessibility child" that has the wrong role so makes the markup invalid, but idk, seems weird to introduce an "invalid" example here!